test/unit/handlers/authorize-handler_test.js

```diff
@@ -87,11 +87,11 @@ describe('AuthorizeHandler', function() {
       };
       const handler = new AuthorizeHandler({ authorizationCodeLifetime: 120, model: model });
 
-      return handler.saveAuthorizationCode('foo', 'bar', 'qux', 'biz', 'baz', 'boz')
+      return handler.saveAuthorizationCode('foo', 'bar', 'qux', 'biz', 'baz', 'boz', 'codeChallenge', 'codeChallengeMethod')
```

Comment:
I just have one request here. Can you make this another unit and keep the original one, please? We should make sure, that the former test is kept the way it is to ensure it won't break if PKCE is not used at all.

Reply:
I added an extra test now instead of updating the old one. Needed to also update in save authorization code to avoid adding empty code challenge properties.